test/sqllogictest/chbench.slt

```diff
@@ -619,9 +619,9 @@ ORDER BY su_nationkey, cust_nation, l_year
 | |   delta %5 %0.(#0) %4.(#2, #1, #3) %3.(#2, #1, #0) %2.(#0, #1) %1.(#0) %0.(#0)
 | |   delta %0 %1.(#3) %2.(#17) %3.(#5, #4) %4.(#0, #1, #2) %5.(#0, #1, #2) %0.(#0)
 | |   delta %0 %5.(#21) %4.(#2, #1, #3) %3.(#2, #1, #0) %2.(#0, #1) %1.(#0) %0.(#0)
-| | demand = (#3, #31, #33, #39, #52, #66, #70)
 | Filter (((#66 = "GERMANY") && (#70 = "CAMBODIA")) || ((#66 = "CAMBODIA") && (#70 = "GERMANY"))), (datetots(#31) <= 2012-01-02 00:00:00), (datetots(#31) >= 2007-01-02 00:00:00)
-| Reduce group=(#3, substr(chartostr(#52), 1, 1), date_part_year_tstz(datetotstz(#39)))
+| Map substr(chartostr(#52), 1, 1), date_part_year_tstz(datetotstz(#39))
```

Comment:
Thinking out loud, but somewhere (maybe further down the pipeline) we probably want to fuse `Map` expressions into `Reduce` in places like these, where there is not a projection that would reduce the data (pushing this map into the `Join` seems not helpful without a projection). Not an issue that needs to be fixed here, but maybe as part of whatever "absorbing" work we end up doing, closer to LIR.